libs/user/src/lib/login/login.component.html

```diff
@@ -88,17 +88,13 @@ <h1>{{ 'login.title' | translate }}</h1>
     </div>
 
     <div class="login_signup">
-      @if (!(isOnboardingDisabled | async)) {
-        <span>{{ 'login.have_account' | translate }}</span>
-        <a
-          [routerLink]="['../signup']"
-          data-cy="create-account"
-          queryParamsHandling="merge">
-          {{ 'login.create_one' | translate }}
-        </a>
-      } @else {
-        {{ 'login.onboarding-disabled' | translate }}
```

Comment:
can `login.onboarding-disabled` be removed ?

Reply:
you're right! let me do that